<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UImagesWidget</class>
 <widget class="QWidget" name="UImagesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameImages">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <layout class="QGridLayout" name="gridLayoutImages">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxShowLegend">
        <property name="text">
         <string>Show legend</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIndChannels">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Ind. channels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonOriginalSize">
        <property name="text">
         <string>Original size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonPropSize">
        <property name="text">
         <string>Prop. size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonTiledSize">
        <property name="text">
         <string>Tiled size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSaveToBMP">
   <property name="text">
    <string>Save to BMP</string>
   </property>
  </action>
  <action name="actionSaveToJPEG">
   <property name="text">
    <string>Save to JPEG</string>
   </property>
  </action>
  <action name="actionSaveAllToBMP">
   <property name="text">
    <string>Save all to BMP</string>
   </property>
  </action>
  <action name="actionSaveAllToJPEG">
   <property name="text">
    <string>Save all to JPEG</string>
   </property>
  </action>
  <action name="actionSelectSource">
   <property name="text">
    <string>Select source</string>
   </property>
  </action>
  <action name="actionAddColumn">
   <property name="text">
    <string>Add column</string>
   </property>
  </action>
  <action name="actionAddRow">
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="actionDeleteColumn">
   <property name="text">
    <string>Delete column</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
